crates/runtime/src/actor/component/bus.rs

```diff
@@ -92,15 +97,23 @@ impl host::Host for Ctx {
     #[instrument(skip(self, payload))]
     async fn call_sync(
         &mut self,
-        target: Option<host::TargetEntity>,
+        target: Option<lattice::WrpcTarget>,
         operation: String,
         payload: Vec<u8>,
     ) -> anyhow::Result<Result<Vec<u8>, String>> {
-        let target = target
-            .map(TryInto::try_into)
-            .transpose()
-            .context("failed to parse target")?;
-        match self.handler.call_sync(target, operation, payload).await {
+        let interface = TargetInterface::from_operation(&operation)?;
```

Comment:
I'm really curious to see how the `wrpc` library is handling the target here, since both this PR and my PR are manually crafting the target interface but we should know exactly what we're calling based on the component import and we shouldn't have to do all the string parsing

Reply:
Yeah `operation` here being a `String` isn't great -- we should come back and fix this to make it a better type (it's technically very similar to `TargetInterface`)

Reply:
OK so this should be tracked in https://github.com/wasmCloud/wasmCloud/issues/1548 now, we should be able to get to and change this later